hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/HermesMetrics.java

```diff
@@ -1,32 +1,13 @@
 package pl.allegro.tech.hermes.common.metric;
 
-import com.codahale.metrics.ConsoleReporter;
-import com.codahale.metrics.Counter;
-import com.codahale.metrics.Gauge;
-import com.codahale.metrics.Histogram;
-import com.codahale.metrics.Meter;
-import com.codahale.metrics.MetricRegistry;
-import com.codahale.metrics.Timer;
-import com.codahale.metrics.graphite.Graphite;
-import com.codahale.metrics.graphite.GraphiteReporter;
-import com.google.common.base.Joiner;
+import com.codahale.metrics.*;
```

Comment:
\[Checkstyle\] ERROR: Using the '\.\*' form of import should be avoided \- com\.codahale\.metrics\.\*\.